plugin/trino-redshift/pom.xml

```diff
@@ -36,7 +26,7 @@
         <dependency>
             <groupId>com.amazon.redshift</groupId>
             <artifactId>redshift-jdbc42</artifactId>
-            <version>2.0.0.1</version>
+            <version>2.0.0.5</version>
```

Comment:
nit:

Can be two commits. One for bumping Redshift driver version (which has a side effect that you no longer need the entry in repositories). Another for making the Confluent one use https.